packages/controllers/src/controllers/ApiController.ts

```diff
@@ -296,11 +295,6 @@ export const ApiController = {
     fetchRecommendedWallets = true,
     fetchNetworkImages = true
   }: PrefetchParameters = {}) {
-    // Avoid pre-fetch if user is already connected as there is no need to fetch wallets in that case
-    if (AccountController.state.status === 'connected') {
-      return Promise.resolve()
-    }
-
```

Comment:
why do we still need wallet icons even afte rconnection ?

Reply:
If it's because you can connect a namespace but not another, we should check for connection on that namespace.